src/Selenium2Library/keywords/_formelement.py

```diff
@@ -180,6 +181,30 @@ def choose_file(self, locator, file_path):
                         % file_path)
         self._element_find(locator, True, True).send_keys(file_path)
 
+    def choose_multiple_files(self, locator,browser="chrome", basePath="", *files):
+        """Inputs the "files" into file input field found by `locator`.
+        This Keyword extend the Choose Files option to allow user to select multiple files
+        (input type=files multiple).
+        The file paths need to be specified for each file.
+        Alternative is use basePath argument and then only files name.
+        The file specified with `file_path` must be available on the same host
+        where the Selenium Server is running.
+        Example:
+        | Choose Multiple Files | my_upload_field | chrome  | /home/user/files/ | trades.csv | trades.xsl | trades.doc |
+        | Choose Multiple Files | my_upload_field | firefox |                   | /folder/trades.csv | /folder1/trades.xsl | /folder3/trades.doc |
+        """
+        fileList = []
+        for file in files:
+            path_to_image = os.path.join(basePath, file)
+            if not os.path.isfile(path_to_image):
+                raise AssertionError("File '%s' does not exist on the local file system" % path_to_image)
+            fileList.append(path_to_image)
+            if browser not in ("googlechrome", "gc", "chrome"):
+                self._element_find(locator, True, True).send_keys(path_to_image)
+        if browser in ("googlechrome", "gc", "chrome"):
+            logging.info(fileList)
```

Comment:
My bad. I've left it inside but my idea was to remove all of the logging, in fact i don't know how much sense has to print the filelist.
Thanks :)